spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/config/SkipperServerPlatformConfiguration.java

```diff
@@ -46,36 +52,59 @@ public DeployerInitializationService deployerInitializationService(DeployerRepos
 		return new DeployerInitializationService(deployerRepository, platforms, resolver);
 	}
 
-	@Bean
 	@Profile("local")
-	public Platform localDeployers(LocalPlatformProperties localPlatformProperties) {
-		List<Deployer> deployers = new ArrayList<>();
-		Map<String, LocalDeployerProperties> localDeployerPropertiesMap = localPlatformProperties.getAccounts();
-		if (localDeployerPropertiesMap.isEmpty()) {
-			localDeployerPropertiesMap.put("default", new LocalDeployerProperties());
+	@Configuration
+	static class LocalPlatformConfiguration {
+		@Bean
+		@Primary
```

Comment:
Why is `@Primary` needed now?

Reply:
Because There are multiple `Platform` instances configured in the context always. The ActuatorService for the `local` profile  gets the deployer from this one.

Reply:
Might be worth adding this as a comment. Anytime I see `@Primary` it makes me "pump my brakes" and ask "why is this here?".